common-lib/src/main/scala/com/gu/mediaservice/model/usage/DigitalUsageMetadata.scala

```diff
@@ -1,24 +1,44 @@
 package com.gu.mediaservice.model
 
+import java.net.URI
 import play.api.libs.json._
+import com.gu.mediaservice.syntax._
 
+sealed trait DigitalUsageMetadata {
+  val url: URI
```

Comment:
What does the url refer to?

Reply:
Its a link to the resource that has used an image, e.g composer article, syndication partner's website, front url.

This is a renaming of `webUrl` from before. It feels a bit redundant as `UsageReference` also has it...

Reply:
A quick investigation shows we generate `UsageReferences`, so we can afford to lose them and just use `DigitalUsageMetadata `. I'd suggest that as a separate PR though.

Reply:
Could lead to a container on front, e.g. https://www.theguardian.com/uk#opinion or to a specific story editing mode in new frontstool that will allow for deep-linking those.

Of course, because of a transient nature of image overrides and because the lack of discovery is still frontstool best security mechanism – I just wanted to mention the above for completeness and pure pedantry.